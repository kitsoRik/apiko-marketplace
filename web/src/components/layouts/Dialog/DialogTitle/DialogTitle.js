import React from "react";

import "./DialogTitle.scss";

const DialogTitle = ({ value, className, ...props }) => {
	return (
		<h2 className={`dialog-title ${className ?? ""}`} {...props}>
			{value}
		</h2>
	);
};

export default DialogTitle;
