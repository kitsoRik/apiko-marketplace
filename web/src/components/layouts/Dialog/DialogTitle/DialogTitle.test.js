import React from 'react';
import { create } from 'react-test-renderer';
import DialogTitle from './DialogTitle';


describe("Dialog", () => {
    it("should render with value", () => {
        const component = create(<DialogTitle value="My custom title" />);
        const json = component.toJSON();

        expect(json).toMatchSnapshot();
    });
    it("should render without value", () => {
        const component = create(<DialogTitle />);
        const json = component.toJSON();

        expect(json).toMatchSnapshot();
    });

    it("should render with className", () => {
        const component = create(<DialogTitle className="my-custom-class-name" />);
        const json = component.toJSON();

        expect(json).toMatchSnapshot();
    });
});