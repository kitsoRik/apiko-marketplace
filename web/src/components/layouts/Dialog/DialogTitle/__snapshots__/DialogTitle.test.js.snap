// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dialog should render with className 1`] = `
<h2
  className="dialog-title my-custom-class-name"
/>
`;

exports[`Dialog should render with value 1`] = `
<h2
  className="dialog-title "
>
  My custom title
</h2>
`;

exports[`Dialog should render without value 1`] = `
<h2
  className="dialog-title "
/>
`;
